### ISSUE TITLE
getaddrinfo ENOTFOUND registry.npmjs.org

### ISSUE THREAD

**Issue by gunta** [rstacruz/pnpm#87]
Still getting this with 0.17 on different packages each time.

```
gulp ^3.8.11                                                      ERROR ✗    
...
getaddrinfo ENOTFOUND registry.npmjs.org registry.npmjs.org:80
GET registry.npmjs.org/deprecated/-/deprecated-0.0.1.tgz
```

**Comment by rstacruz** [comment]
#86, the patch for this, will be out on 0.18. in the mean time, you can try `--concurrency 8` to reduce the concurrency, or just retry the `pnpm install` command where pnpm will happily pick up where it left off.

**Comment by gunta** [comment]
Thank you!

**Comment by gunta** [comment]
Currently it even fails with `--concurrency 2`.

**Comment by zkochan** [comment]
I guess this one was fixed with the pull request

**Comment by andreypopp** [comment]
Still getting this sometimes (but super rarely) on 0.25.0.

**Comment by zkochan** [comment]
ok

**Comment by zkochan** [comment]
maybe the keepalive duration has to be configurable. npm probably has something like that and we can use the same value

**Comment by zkochan** [comment]
I mean this https://github.com/rstacruz/pnpm/blob/master/lib/got.js#L21

**Comment by zkochan** [comment]
I think it should finally be solved since [v0.27.0](https://github.com/rstacruz/pnpm/releases/tag/v0.27.0) which added support for the npm fetch-* configs.

**Comment by zkochan** [comment]
This hopefully has to be fixed from [v0.36.0](https://github.com/rstacruz/pnpm/releases/tag/v0.36.0)

**Comment by zkochan** [comment]
Closing this as pnpm uses the same code and configs npm does for fetching packages from the registry, (since v0.36.0)
